.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+### Description
+
+### What works
+
+### What was not tested
+
+### Test
+Operating system (and linux kernel version): 
+Hardware (GPU/CPU):
+
+### Ready for review
+[ ] Script tested as a regular phoenicis user and working (if you have a problem -> draft and ask for help)
+[ ] json-align and eslint run accorded to the .
+
+Codacy and travis must run perfectly for the PR to be reviewed. You can ping reviewers if you think there is a problem with those tools (use draft for this).
```

Comment:
The last sentence is not clear for me.

Reply:
What happens if travis fails without any reason for example.

Reply:
I see. Nevertheless, I'm not sure the approach is useful because as far as I know Travis and Codacy will not run for draft pull requests. So you would have to set the PR to draft after one of the checks fail. That doesn't make too much sense in my opinion. Apart from that, everybody would have to remove the remark from the description because it doesn't really belong to the description. Therefore I would rather go for the checklist only.

Reply:
I would like to at least keep an info about codacy and travis to be checked because it is really annoying to always remind to someone "Please check travis and codacy".

Reply:
Ok but this could also be a point of the checklist, right?